builder/amazon/ebs/step_create_ami.go

```diff
@@ -66,15 +66,19 @@ func (s *stepCreateAMI) Run(ctx context.Context, state multistep.StateBag) multi
 	ui.Say("Waiting for AMI to become ready...")
 	if err := awscommon.WaitUntilAMIAvailable(ctx, ec2conn, *createResp.ImageId); err != nil {
 		log.Printf("Error waiting for AMI: %s", err)
-		imagesResp, err := ec2conn.DescribeImages(&ec2.DescribeImagesInput{ImageIds: []*string{createResp.ImageId}})
-		if err != nil {
+		imagesResp, imerr := ec2conn.DescribeImages(&ec2.DescribeImagesInput{ImageIds: []*string{createResp.ImageId}})
+		if imerr != nil {
```

Comment:
Should we capture any information from `imerr`? Looking at the internals of `awcommon.WaitUntilAMIAvailable` it wraps the error from calling `DescribeImages` so it might already be covered, but just want to make sure.

Reply:
if there's a failure describing the image, I don't think that's relevant to why the image copy failed.

Reply:
:+1: